app/routers.py

```diff
@@ -388,3 +397,19 @@ async def get_source_tags(
         raise GenericException("This database type is not supported (yet)!")
 
     return vectordb.get_available_labels()
+
+@router.get('/api/get-supabase-keys', response_model=schema.SupabaseKeys)
+def get_supabase_keys():
+    """Returns the supabase keys if set in environment variables"""
+    supabase_url = os.environ.get('SUPABASE_URL')
+    supabase_key = os.environ.get('SUPABASE_KEY')
+
+    if not supabase_url or not supabase_key:
+        return {"supabaseUrl": "", "supabaseKey": ""}
+
+    return {"supabaseUrl": supabase_url, "supabaseKey": supabase_key}
+
+@router.get("/modules/{module_name}")
+async def get_module(module_name: str):
+    """Returns the module file"""
```

Comment:
Modules? Is this connected to supabase?

Reply:
It was my javascript functions, to clean them up out of the html files. But not needed now, so deleted.